vpr/src/pack/cluster.cpp

```diff
@@ -1169,15 +1175,43 @@ static enum e_block_pack_status try_pack_molecule(
 				}
 			}
 			if (block_pack_status == BLK_PASSED) {
-				/* Try to route if heuristic is to route for every atom
-					Skip routing if heuristic is to route at the end of packing complex block
-				*/
-				bool is_mode_conflict = true;
+				/*
+				 * during the clustering step of `do_clustering`, `detailed_routing_stage` is incremented at each iteration until it a cluster
```

Comment:
Fix this formatting to conform to an 80 character maximum.  I recommend pulling comment out of here and putting it somewhere else.

Reply:
Done